### PR TITLE
Add integration test for inference with CSV

### PR DESCRIPTION
**Pull request by laurenyu**
in light of #46, this seemed prudent.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/integration/local/test_mnist_training.py [hunk 1]
```diff
@@ -38,13 +39,22 @@ def test_mnist_training_and_serving(docker_image, sagemaker_local_session, local
     with local_mode.lock():
         try:
             model = mx.create_model(model_server_workers=NUM_MODEL_SERVER_WORKERS)
-            predictor = model.deploy(1, local_instance_type)
+            predictor = _csv_predictor(model, local_instance_type)
             data = numpy.zeros(shape=(1, 1, 28, 28))
             predictor.predict(data)
```
Comment (icywang86rui) [inline test/integration/local/test_mnist_training.py]: should we verify that the return value here is valid?

File: test/integration/local/test_mnist_training.py [hunk 2]
```diff
@@ -38,13 +39,22 @@ def test_mnist_training_and_serving(docker_image, sagemaker_local_session, local
...
         finally:
             mx.delete_endpoint()
 
 
+def _csv_predictor(model, instance_type):
+    predictor = model.deploy(1, instance_type)
+    predictor.content_type = 'text/csv'
+    predictor.serializer = csv_serializer
+    predictor.accept = 'text/csv'
+    predictor.deserializer = None
```
Comment (icywang86rui) [inline test/integration/local/test_mnist_training.py]: why don't we test the deserializer here as well?

Comment (laurenyu) [reply test/integration/local/test_mnist_training.py]: these are Python SDK entities; the Python SDK doesn't have a built-in CSV deserializer

File: test/integration/local/test_mnist_training.py (hunk 1 above)
Comment (laurenyu) [reply test/integration/local/test_mnist_training.py]: added an assert

**Pull request merged**